source/chapters/library.rst

```diff
@@ -347,7 +352,7 @@ a crate, or even the whole library) for tracks that match your search query.
   library. Use the :kbd:`ARROW UP` and :kbd:`ARROW DOWN` keys to scroll in the
   list of results.
 
-.. note:: If the search input field has focus, the Mixxx keyboard shortcuts are
+.. note:: If the search input field has focus, the Mixxx keyboard :term:`shortcuts <shortcuts>` are
```

Comment:
This now feels redundant: I think the shortcuts reference is obsolete here when we have "see :ref:`control-keyboard`", so please remove the change.

I'm sorry this gets confusing with multiple reviewers...

Reply:
Done!